libvast/include/vast/arrow_table_slice.hpp

```diff
@@ -71,8 +71,10 @@ class arrow_table_slice final {
   /// @param slice The encoding-specific FlatBuffers table.
   /// @param parent The surrounding chunk.
   /// @param batch A pre-existing record batch.
+  /// @param schema VAST schema matching the record batch schema.
```

Comment:
The documentation should explain that this argument is optional and exists purely for optimization purposes.

Reply:
It's currently not optional, should I change that?

Reply:
The parameter may not be optional, but the argument semantically is, as you can pass in a default-constrcuted type and the constructor infers it from the batch for you.